libgadget/drift.c

```diff
@@ -13,9 +12,9 @@
 #include "partmanager.h"
 #include "utils.h"
 
-static void
-real_drift_particle(int i, inttime_t ti1, const double ddrift, const double random_shift[3]);
+static void real_drift_particle(int i, inttime_t ti1, const double ddrift, const double BoxSize, const double random_shift[3]);
 
+#if 0
```

Comment:
Disabled because ??
Or perhaps explain when this may need to be reenabled.

Reply:
It is unused and I didn't want to waste time fixing it. I would be happy to delete it too, but I keep it around as an example of how the locking should work.

Reply:
Deleted